src/edu/csus/ecs/pc2/ui/ShadowCompareRunsPane.java

```diff
@@ -333,8 +753,7 @@ private void saveCSVFile() {
                     try {
                         saveFile.createNewFile();
                     } catch (IOException e) {
-                        // TODO Auto-generated catch block
-                        e.printStackTrace();
+                        log.log(Log.SEVERE, "Exception saving file: " + e.getMessage(), e);
```

Comment:
more context in log information that answers: which files was not saved?
Update message to:
"Exception saving judgement comparison CSV file : " + e.getMessage(), e);

Reply:
Done.